spec/controllers/v2/content_items_controller_spec.rb

```diff
@@ -45,6 +45,27 @@
       )
     end
 
+    context "searching a field" do
+      context "when there is a valid query" do
+        let!(:content_item) do
+          FactoryGirl.create(:live_content_item,
+                             base_path: "/foo",
+                             format: "topic",
+                             title: "foo bar")
```

Comment:
Would be better remove "foo" from the title so that there is no overlap between the fields.